<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1019</width>
    <height>817</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(40, 44, 52);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="topBar">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(33, 37, 43);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="leftTopBar">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>14</x>
            <y>10</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../resources/assert/github.svg</pixmap>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>0</y>
            <width>211</width>
            <height>41</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Baskerville Old Face</family>
            <pointsize>14</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>GitCommitBuddy</string>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="rightTopBar">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnMinimize">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string extracomment="Minimize"/>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:none;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../resources/assert/minimize.svg</normaloff>../resources/assert/minimize.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnClose">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="toolTip">
             <string extracomment="Close"/>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:none;
}
</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../resources/assert/x.svg</normaloff>../resources/assert/x.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="mainContent">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="leftMenu">
         <property name="maximumSize">
          <size>
           <width>55</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(33, 37, 43);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="topMenu">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>250</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>300</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="pushButton">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:none;
}

QPushButton:hover{
	border-left: 3px solid rgb(255,255,255);
}


 QPushButton:focus{
    color: rgb(255, 255, 255);
    border-left: 2px solid rgb(255,255,255);	
  }</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resources/assert/home.svg</normaloff>../resources/assert/home.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="autoDefault">
                <bool>false</bool>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_2">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:none;
}

QPushButton:hover{
	border-left: 3px solid rgb(255,255,255);
}


 QPushButton:focus{
    color: rgb(255, 255, 255);
    border-left: 2px solid rgb(255,255,255);	
  }</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resources/assert/git-commit.svg</normaloff>../resources/assert/git-commit.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_3">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:none;
}

QPushButton:hover{
	border-left: 3px solid rgb(255,255,255);
}


 QPushButton:focus{
    color: rgb(255, 255, 255);
    border-left: 2px solid rgb(255,255,255);	
  }</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resources/assert/git-pull-request.svg</normaloff>../resources/assert/git-pull-request.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pushButton_4">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton{
	color: rgb(255, 255, 255);
	border:none;
}

QPushButton:hover{
	border-left: 3px solid rgb(255,255,255);
}


 QPushButton:focus{
    color: rgb(255, 255, 255);
    border-left: 2px solid rgb(255,255,255);	
  }</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset>
                 <normaloff>../resources/assert/layers.svg</normaloff>../resources/assert/layers.svg</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>30</width>
                 <height>30</height>
                </size>
               </property>
               <property name="flat">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="bottomMenu">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="rightContent">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(40, 44, 52);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(33, 37, 43);</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
